keyboards/skdevices/adler/keyboard.json

```diff
@@ -0,0 +1,292 @@
+{
+    "manufacturer": "SKDevices",
+    "keyboard_name": "Adler",
+    "maintainer": "Krzysztof Smosna",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+        "oled": true
+    },
+    "build":{
+        "debounce_type": "asym_eager_defer_pk"
+    },
+    "matrix_pins": {
+        "cols": ["A10", "A9", "A8", "C9", "C8", "C7", "C6", "B15", "B14", "B13", "B12", "B10", "B2", "B1", "B0", "C5", "C4", "A7"],
+        "rows": ["B5", "B4", "B3", "D2", "C12", "C11"]
```

Comment:
Has this been tested on hardware? 

B2 is boot1 and using it can cause issues.  IIRC, A10 can cause issues with getting into the stm32-dfu bootloader.  A9 is vbus sense and has an internal pulldown, IIRC, so may have issues.